src/vttLib/transfer.py

```diff
@@ -0,0 +1,72 @@
+import io
+import logging
+import os
+
+import fontTools
+import fontTools.ttLib
+
+import vttLib
+
+logger = logging.Logger(__name__)
+
+
+def dump_to_file(font: fontTools.ttLib.TTFont, path: os.PathLike):
+    TABLES_TO_DUMP = ("TSI1", "TSI3", "TSI5", "maxp")
+    for table_tag in TABLES_TO_DUMP:
+        if table_tag not in font:
+            raise vttLib.VTTLibArgumentError(
+                "Table '%s' not found in input font" % table_tag
+            )
+
+    vttLib.normalize_vtt_programs(font)
+    font.saveXML(path, tables=TABLES_TO_DUMP)
+
+
+def merge_from_file(font: fontTools.ttLib.TTFont, path: os.PathLike):
+    if "TSI0" not in font:
+        font["TSI0"] = fontTools.ttLib.newTable("TSI0")
+    if "TSI2" not in font:
+        font["TSI2"] = fontTools.ttLib.newTable("TSI2")
+    font.importXML(path)
```

Comment:
If you dump the whole maxp table in the ttx file and then import it as is, it will overwrite other fields that you do not want to change like numGlyphs (a UFO may have additional glyphs or less glyphs since the last dump). That is why we were dumping only a subset of fields to a plist and merging it ourselves without using ttx machinery, which only supports dumping/merging whole tables.